README.md

```diff
@@ -335,14 +340,17 @@ APP_ID_IOS = org.mozilla.ios.FirefoxVPN
 NETEXT_ID_IOS = org.mozilla.ios.FirefoxVPN.network-extension
 ```
 
-6. Generate the Xcode project using our script (and an optional adjust token):
+6. Make sure qmake is available by setting the environment variable QT_IOS_BIN:
+
+```
+export QT_IOS_BIN=/Users/[username]/Qt/6.2.4/ios/bin
+```
+
+Then generate the Xcode project using our script (and an optional adjust token):
+
 ```bash
-./scripts/macos/apple_compile.sh ios [--adjust <adjust_token>]
+./scripts/macos/apple_compile.sh ios [--adjust <adjust_token>] -q ~/Qt/6.2.4/macos/bin
 ```
-(If you get an error like `Step 7: Generate translation resources...
-sh: /Users/[username]/Qt/6.2.4/ios/bin/lconvert: No such file or directory`
-you may need to provide the macOS Qt bin folder through the -q flag:
-`./scripts/macos/apple_compile.sh ios -q ~/Qt/6.2.4/macos/bin`)
```

Comment:
Seems easier just to tell people to use the -q flag.

Reply:
👍🏻